src/scripts/items/PokemonItem.ts

```diff
@@ -35,6 +35,16 @@ class PokemonItem extends CaughtIndicatingItem {
 
         const pokemonID = PokemonHelper.getPokemonByName(pokemonName).id;
         App.game.party.gainPokemonById(pokemonID, shiny, true);
+        // Only for statistics
+        for (let i = 0; i < amt - 1; i++) { // -1 because gainPokemonById will add 1 to statistics
+            const gender = PokemonFactory.generateGenderById(pokemonID);
+            // Add to shiny statistics depending of the number of shinies obtained
```

Comment:
Can we mash this for loop together with the shiny for loop? They are doing basicly the same, so might as well add them together

Reply:
Done.